.github/workflows/srtool-build.yml

```diff
@@ -0,0 +1,84 @@
+name: Srtool build
+
+on:
+  release:
+    types: [published]
```

Comment:
But this is a big change. When we publish the release we want to have the WASM already built.

Reply:
We need to run this whenever we used to run it before, which I believe is on merges into `develop` and `master`. We need it even more now since we are using Chopsticks and we need to have the WASM available for testing pre-release.

Reply:
Trigger as before 
```  push:
    branches:
      - develop
      - master
      - 'refs/tags/[0-9]+.[0-9]+.[0-9]+*'
```

Reply:
I guess you'll remove the `ci-cache` branch in an upcoming PR?